docker/hq-compose.yml

```diff
@@ -110,14 +110,14 @@ services:
       - ./files/elasticsearch_5.yml:/usr/share/elasticsearch/config/elasticsearch.yml:rw
 
   zookeeper:
-    image: wurstmeister/zookeeper
+    image: zookeeper:3.7
     ports:
       - "2181:2181"
     volumes:
-      - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.13/data
+      - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.7/data
 
   kafka:
-      image: wurstmeister/kafka:2.13-2.6.0
+      image: bitnami/kafka:3.2
```

Comment:
I assume this is a trusted/vetted container source?

Reply:
Yeah they are a verified publisher with 100+ million downloads of that specific image, as well as lots of downloads of other images. Bitnami is owned by VMWare fwiw.